images/Dockerfile.ubuntu.certified.aptrepo.readytoserve

```diff
@@ -7,5 +7,6 @@ RUN apt-get install -y openssh-server sudo
 RUN rm /etc/ssh/ca-user-certificate-key.pub
 EXPOSE 22
 EXPOSE 8000
+ARG CACHEBUST=1
```

Comment:
Fix: ssh to apt_repo container fails because it has the old ca-user-certificate-key.